sql/wsrep_mysqld.cc

```diff
@@ -1390,7 +1394,22 @@ static void wsrep_push_warning(THD *thd,
                  ha_resolve_storage_engine_name(hton),
                  tables->db.str, tables->table_name.str);
     break;
-
+  case WSREP_EXPERIMENTAL:
+    push_warning_printf(thd, Sql_condition::WARN_LEVEL_WARN,
+                        ER_ERROR_DURING_COMMIT,
+                        "WSREP: Replication of non-transactional engines are experimental. "
+                        "Storage engine %s for table '%s'.'%s' is "
```

Comment:
Replication of non-transactional engines *is* experimental

Reply:
Fixed.